frontend/public/locales/en/index.json

```diff
@@ -0,0 +1,23 @@
+{
+  "page-title": "Home",
+  "welcome": "Welcome {{firstName}} {{lastName}}",
+  "breadcrumbs": {
+    "home": "Home"
+  },
+  "personal-info": "Personal information",
+  "personal-info-desc": "In this section, you can see your personal information.",
+  "update-info": "Update personal information",
+  "update-info-desc": "In this section, you can your personal information.",
+  "upload": "Upload document for CDCP",
+  "upload-desc": "In this section, you can your upload documents.",
+  "view-letters": "View my letters",
+  "view-letters-desc": "In this section, you can see any letters that were sent for you.",
+  "view-CDCP": "View my CDCP messages",
+  "view-CDCP-desc": "In this section, you can see any messages that were sent to you.",
```

Comment:
**Nitpick:** the convention is to use kebab-case for these keys. In this case, it would be `view-cdcp` and `view-cdcp-desc`.

Reply:
Insta-resolving this so the PR can be merged. We can change the i18n key in a future PR.

Reply:
I'll make the change while resolving conflicts.